app/src/main/java/ai/elimu/vitabu/ui/storybook/ChapterFragment.java

```diff
@@ -189,7 +188,7 @@ public void onViewCreated(@NonNull View view, @Nullable Bundle savedInstanceStat
         super.onViewCreated(view, savedInstanceState);
 
         // Add button for initializing Text-to-Speech (TTS)
-        final String finalChapterText = chapterText;
+        final String[] finalChapterText = chapterText;
```

Comment:
`finalChapterText` might be mistaken for meaning "the final text of this chapter", so perhaps we should rename this variable?